### ISSUE TITLE
Clickable Area

### ISSUE THREAD

**Issue by edurocks** [heremaps/here-android-sdk-examples#388]
Good Morning.
 Is there a way to increase the clickable area of the marker on the map? Because right now I have to click exactly on the marker if I click a little bit outside it doesn´t catch the click. Is there a way to achieve that? Thanks.

**Comment by NazarKacharaba** [comment]
There is no such API in the SDK 3.x, but you could try adding transparent edges in your image, so the visual size of the marker will be the same, but the clickable area will be bigger(because icon's width and height will be bigger).

**Comment by edurocks** [comment]
My Image is in png. Do you mean increase the width and height of the image with transparency and keep the image itself with the same size?

**Comment by NazarKacharaba** [comment]
Yes. The final size of the PNG will be bigger. Size of the visible area of the image will be the same.

**Comment by edurocks** [comment]
Understood. Yes this might solve the problem. I will try. My designer will do this idea. Thanks. Maybe you can close this issue

**Comment by ruwaydaalshowiman** [comment]
I have a problem, how can I set a click listener on a (Marker labeled) or how can I let the marker be clickable
